src/app_task.cpp

```diff
@@ -620,7 +658,9 @@ void AppTask::ZigbeeEventHandler(ZigbeeShell * shell, ZigbeeShell::Event_t event
 		for (auto &light : Lights)
 		{
 			if (!strcmp(light.GetName(), "none")) {
-				AddDeviceEndpoint(&light, &bridgedLightEndpoint, DEVICE_TYPE_LO_ON_OFF_LIGHT);
+				AddDeviceEndpoint(&light, &bridgedLightEndpoint,
```

Comment:
Considering the fact this method may fail and return -1, I think the returned value should be somehow handled.

Reply:
done.